src/logic/logic.js

```diff
@@ -33,9 +33,9 @@ function assocSymbol(field) {
     case 'PERFECT_KEY':
       return '1';
     case 'PRIMARY_KEY':
-      return '1';
+      return '0/1';
     default:
-      return 'n';
+      return '0..n';
```

Comment:
Did we decide if there was an existing description of these out there that we can base it on? I.e. [UML](https://www.uml-diagrams.org/multiplicity.html) defines the notation `0..*` or simply `*`.

Reply:
We did investigate the page:
https://www.uml-diagrams.org/multiplicity.html

The * was preferred but hard to distinguish in the UI. We did some testing on alternative asterisks 
https://unicode-search.net/unicode-namesearch.pl?term=ASTERISK
but ended up in using the more descriptive 0..n. Could however try 0..* to better reflect the UML specification.

Reply:
I'm not really fond of 0..n, and would prefer 0..*. But ofc it should be easy to see in the UI as well.